### ISSUE TITLE
Improve OWID's accessibility lighthouse score

### ISSUE THREAD

**Issue by ikesau** [owid/owid-grapher#1595]
## Core problem

Across various pages for both desktop and mobile, we're "leaving accessibility on the table" by having buttons without roles, insufficient contrasts, non-hierarchical headings, and missing attributes.

On Desktop, many of our pages hover around 85-90%

On Mobile, most pages are closer to 80%

## Proposed solution

Dedicate a day and a dev to running WebAIM's [WCAG Audit](https://wave.webaim.org/report#/ourworldindata.org) and Google Lighthouse to fix as many of the easy issues as possible.

## Alternatives

A more robust and exhaustive [personal WCAG 2.1 Level AA audit](https://web.dev/how-to-review/) could also be done.

## Context

![image](https://user-images.githubusercontent.com/11844404/184706365-97882e99-be96-4244-b77d-ab3a966eebe7.png)

**Comment by larsyencken** [comment]
Chatted to Daniel B, possible we could pick this up just before Sicily, if it doesn't get done in a cooldown before then.

**Comment by larsyencken** [comment]
Proposed in Dev slack channel to do it together next cooldown

**Comment by toni-sharpe** [comment]
_Not_ taking ownership until instructed but my Lighthouse scores on my demo site are 98-100% so this is something I could pick at without the pressure of cycle deadlines and while looking for work and beyond, incremental, potentially tiny little changes.

@larsyencken let me know if this fits with you.

No more tags now either :-) this would keep me out of the way and busy improving things,.

**Comment by ikesau** [comment]
Worked on this via https://github.com/owid/owid-grapher/pull/4131

This will be a perennial thing to work on
